docs/guides/trace/bind.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: 'The "bindsnoop" gadget'
+title: 'The "trace bind" gadget'
```

Comment:
This change is missing in some guides, for example for DNS, socket-collector and others. Could you please update them?

Reply:
I am not convinced with the new title as I find it odd.
So, I would like to have other opinions before doing the whole change.

Reply:
Yeah, me neither. What about "The bind tracer gadget", "The seccomp-policy advisor", "The socket snapshot" ... etc. BTW, I just realize that some categories are nouns and other verbs. For instance, we have "advisor" but "audit" instead of "auditor". Same for trace, snapshot. It is odd.

Reply:
> I just realize that some categories are nouns and other verbs. For instance, we have "advisor" but "audit" instead of "auditor". Same for trace, snapshot. It is odd.

I totally agree.
I can rename `advisor` to `advise` , so we would have all verbs but I am not 100% convinced by `kubectl gadget advise seccomp-policy`.

Reply:
I pushed the advisor renaming in a fixup commit, so if we do not agree on it, I would just need to drop it.